src/typescript-service.ts

```diff
@@ -32,11 +34,30 @@ export default class TypeScriptService {
         this.root = root;
         this.projectManager = new pm.ProjectManager(root, strict, connection);
 
+        this.fetchDefinitelyTypedRepo();
+
         //initialize providers 
         this.exportedSymbolProvider = new ExportedSymbolsProvider(this);
         this.externalRefsProvider = new ExternalRefsProvider(this);
     }
 
+    /**
+     * Fetch repository from https://github.com/DefinitelyTyped/DefinitelyTyped into /tmp/DefinitelyTyped/
+     */
+    fetchDefinitelyTypedRepo() {
+        fs.exists('/tmp/DefinitelyTyped', (exists) => {
```

Comment:
If the clone fails, then will subsequent operations attempt to reclone? Cloning from GitHub probably fails 2% of the time, and it would be bad if the clone failed when this server first started up and we have no way of knowing (other than checking elevated error rates) or recovering.

Will an error be returned that can be associated with the particular session that it occurred in, or will we have to look at the server logs? (Seems like the latter.)